tests/regression/lp-1828354/task.yaml

```diff
@@ -0,0 +1,206 @@
+summary: mount event propagation works inside snapd-created tmpfs
+# NOTE: The limitation comes from availability of Python 3.7 for mountinfo-tool
```

Comment:
It is temporary. I will make it compatible with more systems tomorrow. It simply started with 3.7 features because I had not realized those are 3.7 features. It should work everywhere and will do so.

Reply:
This is now done in https://github.com/snapcore/snapd/pull/6940 -- curiously it did find an issue in 16.04 core systems. I will investigate why next week.